pom.xml

```diff
@@ -7,13 +7,13 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.microfocus.octane.plugins</groupId>
     <artifactId>jira-octane-quality-insight-plugin</artifactId>
-    <version>2023.0.0</version>
+    <version>24.4</version>
     <organization>
-        <name>Micro Focus ALM Octane</name>
+        <name>OpenText ValueEdge</name>
```

Comment:
I think it should be only OT
+ url changed to OT org

Reply:
Changing link 
domain is still microfocus, but will now point to value edge